bridge/README.md

```diff
@@ -75,6 +75,17 @@ Per default login attempts are throttled to 10 requests within 60 minutes. This
 - `REQUESTS_WITHIN_TIME` - how many login attempts in which timespan `REQUEST_TIME_LIMIT` (in minutes) are allowed per IP.
 - `CLEAN_BUCKET_INTERVAL` - the interval (in minutes) the saved login attempts should be checked and deleted if the last request of an IP is older than `REQUEST_TIME_LIMIT` minutes. Default is 60 minutes.
 
+### Setting up login via OpenID
+To set up a login via OpenID you have to register an application by the identity provider you want in order to get an ID (`CLIENT_ID`) and a secret (`CLIENT_SECRET`).
```

Comment:
I am not sure but I think it would be helpful to add some additional information here. E.g.: for which grant types/flows is this working. Refresh token enabled/disabled. Is this a public or confidential client a.s.o.